man/import.q.concourse.Rd

```diff
@@ -2,19 +2,22 @@
 \alias{import.q.concourse}
 \title{Q methodology: import concourse of Q items}
 \description{
-  Imports full set of items (concourse) from a directory of *.tex files, including possible translations.
+  Imports full set of items (concourse) from a directory of .TEX files, including possible translations.
 }
 \usage{
-  import.q.concourse(q.concourse.dir, languages)
+  import.q.concourse(q.concourse.dir, languages=NULL)
 }
 
 \arguments{
   \item{q.concourse.dir}{
-    a directory of folders per language, with all full q items as *.tex files in each language folder, with files named as the same meaningful item short handle (e.g. \code{happy-feeling.tex}) across all languages.
+    A directory of individual .TEX files named as meaningful item short handles (e.g. \code{happy-feeling.tex}).
+    If \code{languages} are specified, a directory with folders per language, with all full Q items as individual .TEX files in each language folder, with files named as the \emph{same} meaningful item short handle (e.g. \code{happy-feeling.tex}) across all languages.
```

Comment:
`If \code{languages} are specified, the directory should contain one folder per language, with all full Q items as individual .TEX files in each language folder. The files within each language folder should be named using the \emph{same} item handle (e.g. \code{happy-feeling.tex}) across all languages.`